fibonacci/script.js

```diff
@@ -0,0 +1,34 @@
+// The Fibonacci sequence is a series of numbers in which each number is the sum of the two preceding ones. 
+// It typically starts with 0 and 1.
+
+function fibonacci(num) {
+    let result = [0, 1];
+
+    for (let i = 2; i <= num; i++) {
+        const prevNum1 = result[i -1];
+        const prevNum2 = result[i -2];
```

Comment:
```suggestion
        const prevNum1 = result[i - 1];
        const prevNum2 = result[i - 2];
```
just a suggestion to prettify a little